doc/api.yml

```diff
@@ -1072,6 +1291,99 @@ components:
       properties:
         main_teacher:
           $ref: '#/components/schemas/Teacher'
+    Order:
+      type: string
+      enum:
+        - ASC
+        - DESC
+
+    ExamInfo:
+      type: object
+      properties:
+        title:
+          type: string
+        date:
+          type: string
+          format: date-time
+    Exam:
+      allOf:
+        - $ref: '#/components/schemas/ExamInfo'
+      properties:
+        id:
+          type: string
+        coefficient:
```

Comment:
Why coefficient is not a a part of ExamInfo ?

Then, why do not you directly use Exam inside ExamDetail ?

You can inherit the ID from Exam.

Reply:
@hei-ryan you said here that an ExamInfo is composed of the exam's title and date.
![Uploading image.png…]()